File: app/src/main/java/cz/kutner/comicsdb/activity/MainActivity.java
package cz.kutner.comicsdb.activity;

import android.os.Bundle;

import cz.kutner.comicsdb.R;
import cz.kutner.comicsdb.fragment.ComicsListFragment;

public class MainActivity extends AbstractActivity {
    public final static String COMICS_ID = "cz.kutner.comicsdbclient.comicsdbclient.comics_id";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .add(R.id.container, new ComicsListFragment())
                    .commit();
        }
    }
}

File: app/src/main/kotlin/cz/kutner/comicsdb/main/MainActivity.kt
package cz.kutner.comicsdb.main

import android.os.Bundle
import androidx.activity.compose.setContent
import androidx.activity.ComponentActivity
import cz.kutner.comicsdb.ui.theme.ComicsDBTheme

class MainActivity : ComponentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            ComicsDBTheme {
                MainScreen()
            }
        }
    }
}
